src/framework/components/camera/component.js

```diff
@@ -29,157 +32,204 @@ import { Component } from '../component.js';
+ * @property {pc.Color} clearColor The color used to clear the canvas to before the
+ * camera starts to render. Defaults to [0.75, 0.75, 0.75, 1].
+ * @property {boolean} clearColorBuffer If true the camera will clear the color buffer
+ * to the color set in clearColor. Defaults to true.
  * @property {boolean} clearDepthBuffer If true the camera will clear the depth buffer.
- * @property {boolean} clearStencilBuffer If true the camera will clear the stencil buffer.
- * @property {pc.Vec4} rect Controls where on the screen the camera will be rendered in normalized screen coordinates.
+ * Defaults to true.
+ * @property {boolean} clearStencilBuffer If true the camera will clear the stencil
+ * buffer. Defaults to true.
+ * @property {number} farClip The distance from the camera after which no rendering
+ * will take place. Defaults to 1000.
+ * @property {number} fov The field of view of the camera in degrees. Usually this is
+ * the Y-axis field of view, see {@link pc.CameraComponent#horizontalFov}. Used for
+ * {@link pc.PROJECTION_PERSPECTIVE} cameras only. Defaults to 45.
+ * @property {boolean} horizontalFov Set which axis to use for the Field of View
+ * calculation. Defaults to false.
+ * @property {number} nearClip The distance from the camera before which no rendering
+ * will take place. Defaults to 0.1.
+ * @property {number} orthoHeight The half-height of the orthographic view window (in
+ * the Y-axis). Used for {@link pc.PROJECTION_ORTHOGRAPHIC} cameras only. Defaults to 10.
+ * @property {number} priority Controls the order in which cameras are rendered. Cameras
+ * with smaller values for priority are rendered first. Defaults to 0.
+ * @property {pc.Vec4} rect Controls where on the screen the camera will be rendered in
+ * normalized screen coordinates. Defaults to [0, 0, 1, 1].
  * @property {pc.Vec4} scissorRect Clips all pixels which are not in the rectangle.
- * The order of the values is [x, y, width, height].
- * @property {pc.PostEffectQueue} postEffects The post effects queue for this camera. Use this to add or remove post effects from the camera.
- * @property {boolean} frustumCulling Controls the culling of mesh instances against the camera frustum, i.e. if objects outside of camera should be omitted from rendering.
- * If true, culling is enabled.
- * If false, all mesh instances in the scene are rendered by the camera, regardless of visibility. Defaults to false.
- * @property {pc.callbacks.CalculateMatrix} calculateTransform Custom function you can provide to calculate the camera transformation matrix manually. Can be used for complex effects like reflections. Function is called using component's scope.
+ * The order of the values is [x, y, width, height]. Defaults to [0, 0, 1, 1].
+ * @property {pc.PostEffectQueue} postEffects The post effects queue for this camera.
+ * Use this to add or remove post effects from the camera.
+ * @property {boolean} frustumCulling Controls the culling of mesh instances against

```

Comment:
defaults to false?!

Reply:
It did previously - I've just kept existing behavior. We can flip it to true. In actual fact, if you create a new camera component in the Editor, it defaults to `true`. So this discrepancy is a bit weird and should probably be changed.